VideoLocker/src/main/java/org/edx/mobile/task/GetCommentListTask.java

```diff
@@ -10,17 +10,25 @@
 
     String threadId;
     int page = 1;
+    boolean isQuestionType;
+    boolean shouldGetEndorsed;
 
-    public GetCommentListTask(Context context, String threadId, int page) {
+    public GetCommentListTask(Context context, String threadId, int page, boolean isQuestionType,
```

Comment:
Having 2 constructors seems to be a better option.

Not a fan of sending unnecessary objects/data to a class.